### PR TITLE
Properly unreg's turf changed when space openspace is deleted

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Openspace tracks the turf below for starlight purposes The trouble is if a turf is replaced by another, all its signal registers still persist

I forgot about this, so we'd get runtimes when a /turf/open/space/openspace was replaced, and the turf below it changed

## Why It's Good For The Game

Less random runtimes

### CODE REVIEW

File: code/game/turfs/open/space/space.dm [hunk 1]
```diff
@@ -261,6 +261,12 @@
 	. = ..()
 	AddElement(/datum/element/turf_z_transparency)
 
+/turf/open/space/openspace/Destroy()
+	. = ..()
+	// Signals persist through destory, GO HOME
```
Comment (Toastgoats) [inline code/game/turfs/open/space/space.dm:266]: This user put a typo in his comment! Laugh at this user!

Comment (LemonInTheDark) [inline code/game/turfs/open/space/space.dm:266]:
```suggestion
	// Signals persist through destroy, GO HOME
```

**Comment by Jolly-66** [comment]
Does this fix the stupid issue I keep seeing downstream and that Ducks tried to fix a few months ago?

**Comment by Toastgoats** [comment]
> Does this fix the stupid issue I keep seeing downstream and that Ducks tried to fix a few months ago?

Possibly, this has something to do with tram checks failing randomly over and over because of some turf issue, and is somehow maybe related to adding new shuttles.

**Comment by LemonInTheDark** [comment]
> Does this fix the stupid issue I keep seeing downstream and that Ducks tried to fix a few months ago?

I don't believe so, tho I may have already fixed that do you remember the details?

**Comment by JohnFulpWillard** [comment]
Would this fix https://github.com/tgstation/tgstation/actions/runs/5442853044/jobs/9898922032

**Comment by Jolly-66** [comment]
> I don't believe so, tho I may have already fixed that do you remember the details?

I'll DM you on Discord when I'm done work.

**Comment by LemonInTheDark** [comment]
> Would this fix https://github.com/tgstation/tgstation/actions/runs/5442853044/jobs/9898922032

It should yes

> Also there's a previous iteration of this PR, dunno how aware of it you are: #75871

I was not, we should have merged that

**Comment by san7890** [comment]
mark ready for review when ci is passing

**Comment by LemonInTheDark** [comment]
thank you @out-of-phaze for the slight steal on where to go with this

**Comment by out-of-phaze** [comment]
> Does this fix the stupid issue I keep seeing downstream and that Ducks tried to fix a few months ago?

If it's like my version of the PR, it hopefully will.

**Pull request merged**